rfcs/text/0013-multiple-files.md

```diff
@@ -0,0 +1,220 @@
+# 0013: Multiple files in an event
+
+<!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
+
+- Stage: **2 (Draft)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
```

Comment:
```suggestion
- Stage: **1 (Draft)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
```

Under the [new stages](https://elastic.github.io/ecs/stages.html) (with stage 4 removed), `Draft` would be considered stage 1.